go.mod

```diff
@@ -39,6 +39,7 @@ require (
 )
 
 replace (
+	github.com/projectcalico/api => github.com/coutinhop/api v0.0.0-20211029224736-72ed23299ba3
```

Comment:
We should point this to our API once https://github.com/projectcalico/api/pull/40 is merged

Reply:
This still needs updating.